evm-adapters/testdata/CheatCodes.sol

```diff
@@ -186,6 +186,21 @@ contract CheatCodes is DSTest {
         prank.bar(address(this));
     }
 
+    function testPayable() public {
+        Prank prank = new Prank();
+        address new_sender = address(1337);
+        hevm.deal(new_sender, 10 ether);
+        
+        hevm.prank(new_sender);
+        prank.payableBar{value: 1 ether}(new_sender);
+        assertEq(new_sender.balance, 9 ether);
```

Comment:
want to check `address(this)` balance as well?

Reply:
for sure